content/en/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md

```diff
@@ -7,6 +7,13 @@ slug: sig-usability-spotlight-2021
 
 **Author:** Kunal Kushwaha, Civo
 
+{{< note >}}
+SIG Usability, which is featured in this Spotlight blog, has been deprecated and is no longer active.
+As a result, the links and information provided in this blog post may no longer be valid or relevant.
+Should there be renewed interest and increased participation in the future, the SIG may be revived. However, for the time being, it will be inactive as per the Kubernetes community policy.
+We encourage you to explore other [SIGs](https://github.com/kubernetes/community/blob/master/sig-list.md#special-interest-groups) and resources available on the Kubernetes website to stay up-to-date with the latest developments and enhancements in Kubernetes.
```

Comment:
There's one author (Kunal), so who does “we” refer to here?

Reply:
In this context, "we" refers to the Kubernetes community and not the author of the blog post. It's common for community announcements, updates to use "we" to represent the collective voice of the organization providing the information. In this case, "we" includes the community folks who are informing readers about the deprecation of SIG Usability and encouraging them to explore other resources and SIGs within the Kubernetes community.

Reply:
in the context of Note

Reply:
> In this context, "we" refers to the Kubernetes community and not the author of the blog post. It's common for community announcements, updates to use "we" to represent the collective voice of the organization providing the information. In this case, "we" includes the community folks who are informing readers about the deprecation of SIG Usability and encouraging them to explore other resources and SIGs within the Kubernetes community.

That's fine, but: don't explain this only to the reviewers. Please make it clear to readers too.